compiler/rustc_monomorphize/src/collector/abi_check.rs

```diff
@@ -0,0 +1,111 @@
+use rustc_abi::Abi;
```

Comment:
You can import this from `rustc_target::spec::abi` and then avoid adding a new dependency.

Reply:
Unless I am mistaken, this seems to be a different ABI - in particular it seems to be about the ABI used for a function (i.e. fastcall vs C), while what I want is the ABI used for a parameter (i.e. Vector).

Reply:
That's `rustc_target::abi::Abi` :-)